addons/website/views/snippets/s_cta_mockups.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_cta_mockups" name="Call to Action Mockups">
+    <section class="s_cta_mockups o_cc o_cc2 pt64 pb64">
+        <div class="container">
+            <div class="row o_grid_mode" data-row-count="9">
+                <div class="o_grid_item o_grid_item_image g-height-9 g-col-lg-6 col-lg-6" style="grid-area: 1 / 6 / 10 / 13; z-index: 1;">
+                    <img src="web_editor/image_shape/website.s_cta_mockups_default_image/web_editor/devices/macbook_front.svg" class="img img-fluid" data-shape="web_editor/devices/macbook_front" data-original-mimetype="image/jpeg" data-file-name="s_cta_mockups.jpg"/>
+                </div>
+                <div class="o_grid_item g-height-5 g-col-lg-4 col-lg-4" style="grid-area: 3 / 1 / 9 / 5; z-index: 2;">
+                    <h3>50,000+ companies trust Odoo.</h3>
+                    <p class="lead">Join us and make your company a better place.</p>
+                    <a t-att-href="cta_btn_href" class="btn btn-primary btn-lg"><t t-esc="cta_btn_text">Contact us</t></a>
```

Comment:
Could you use t-out instead of t-esc? We have a task that replace all the t-esc by t-out in qweb that will be merged soon... Thx!